pkg/cloud/services/eks/addons.go

```diff
@@ -214,6 +214,6 @@ func convertConflictResolution(conflict ekscontrolplanev1.AddonResolution) *stri
 	case ekscontrolplanev1.AddonResolutionOverwrite:
 		return aws.String(eks.ResolveConflictsOverwrite)
 	default:
-		return nil
+		return aws.String(eks.ResolveConflictsOverwrite)
```

Comment:
Is it worth simplifying this function?

```
func convertConflictResolution(conflict ekscontrolplanev1.AddonResolution) *string {
	if conflict == ekscontrolplanev1.AddonResolutionNone {
		return aws.String(eks.ResolveConflictsNone)
	}
	return aws.String(eks.ResolveConflictsOverwrite)
}
```

Reply:
Sure why not. I don't think there will ever be a third option. So it makes sense to simplify it.

Reply:
Yeah that's what I was thinking but wasn't 100% sure if more options could be possible. Either way, code change would be needed if they ever are so can change it back then if needed.